prime-angular-frontend/src/app/modules/site-registration/pages/remote-user-page/remote-user-page.component.html

```diff
@@ -71,15 +71,6 @@
                                             (remove)="removeCertification($event)"></app-college-certification-form>
 
           </ng-container>
-
-          <button mat-button
```

Comment:
So this leads to a discussion about when we are asked to remove features (in this case only one certification for a remote user) should we remove the supporting code and refactor.

Since the next person coming into the component may not realize the form state and controller have a lot of code that no longer reflects the state of the app to the user. 

I think in this sprint we have time to spend some of it to refactor this so that the page only supports one certification.

Reply:
Is this being addressed?

Reply:
Apologies I didn't join Zoom chat to talk about this.  I tried addressing but haven't yet completed.  Instead I found a couple of other defects that exist in dev.pharmanetenrolment.gov.bc.ca; see PRIME-2050